### PR TITLE
Move issues to Github forms

### PR DESCRIPTION
**Pull request by McGiverGim**
Same change that in the firmware project: https://github.com/betaflight/betaflight/pull/11322

GitHub now has a new feature to use forms in the issues tab.

The users have a lot of problems to fill the template in a correct way. Now it will be a lot easier.

This is the result:
image

image

### CODE REVIEW

**Review by KarateBrot — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature-request.yml
Comment (Asizon) [inline .github/ISSUE_TEMPLATE/feature-request.yml:2]: Double "for" :)

File: .github/ISSUE_TEMPLATE/bug-report.yml
Comment (Asizon) [inline .github/ISSUE_TEMPLATE/bug-report.yml:62]: No final empty line.

**Review by Asizon — CHANGES_REQUESTED** [review]
This looks so good!

**Review by Asizon — CHANGES_REQUESTED** [review]
This looks so good!

File: .github/ISSUE_TEMPLATE/feature-request.yml
Comment (Asizon) [reply .github/ISSUE_TEMPLATE/feature-request.yml:2]: Maybe best: request a new feature idea for bet conf??

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: .github/ISSUE_TEMPLATE/feature-request.yml
Comment (McGiverGim) [reply .github/ISSUE_TEMPLATE/feature-request.yml:2]: The double 'for' was in the original file too XD

Fixed.

File: .github/ISSUE_TEMPLATE/bug-report.yml
Comment (McGiverGim) [reply .github/ISSUE_TEMPLATE/bug-report.yml:62]: Fixed.

File: .github/ISSUE_TEMPLATE/feature-request.yml
Comment (Asizon) [reply .github/ISSUE_TEMPLATE/feature-request.yml]: yes, I see

**Review by Asizon — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
